web/app/css/styles.css

```diff
@@ -12,9 +12,9 @@
   --coldgrey: #c9c9c9;
   --neutralgrey: #828282;
   --silver: #BDBDBD;
-  --green: #27AE60;
-  --siennared: #EB5757;
```

Comment:
can we change the name of `siennared` to `red`? While it is a rather arbitrary name, I doubt it matches the original sianna red now anyway :)